api/v1alpha1/ionoscloudmachine_types.go

```diff
@@ -168,6 +168,18 @@ type IonosCloudMachineSpec struct {
 	//+optional
 	FailoverIP *string `json:"failoverIP,omitempty"`
 
+	// LanID is the ID of the LAN if an existing LAN is required.
+	// a typical use-case is when an existing LAN has a Failover Group and the FailoverIP configured.
+	// This field is used when the Failover Group is created in another LAN.
+	//
+	// If the machine is a control plane machine, this field will not be taken into account.
+	// +optional
+	// +immutable
```

Comment:
I don't see this tag in the controller-gen help output. It also doesn't seem to have an effect on the generated YAML. Is this defined somewhere?

Reply:
It's a tag we use, for user readability.